about.md

```diff
@@ -19,7 +19,7 @@ Im Folgenden wollen wir Dir näher bringen was wir hier machen und hoffentlich D
 Ab dem Sommersemester 2022 finden unsere Treffen wieder in Präsenz statt. Immer donnerstags ab 19:00 Uhr im Infobau 50.34, Raum -120.
```

Comment:
Should we change the reference to room -120 to also include -118 just as in https://github.com/kitctf/www/pull/23/files#diff-d6b71de01a710fb7dced2eaffdc7d2605e61f4cd318212668c3f62fc01880699?